addon/components/frost-fixed-table.js

```diff
@@ -429,6 +417,28 @@ export default Component.extend({
     }
   },
 
+  setMinimumCellWidths (bodySelector, numColumns, accountForSelection) {
+    for (let pos = 1; pos <= numColumns; ++pos) {
+      const curBodyColumn = this.$(`${bodySelector} .frost-table-row .frost-table-row-cell:nth-child(` +
```

Comment:
in this case I'm getting the nth column of EACH row. Is there a better way to do this?